src/components/Buttons.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import ButtonCard from "./ButtonCard";
+import "../App.css";
+
+import {
+  Card,
+  CardImg,
+  CardText,
+  CardBody,
+  CardTitle,
+  CardSubtitle,
+  Button,
+} from "reactstrap";
+
+export default (props) => {
+  return (
+    <div>
+      <main>
+        <div className="container">
+          <Card>
+            <CardImg
+              top
+              className="cardImg"
+              src="https://www.history.com/.image/ar_16:9%2Cc_fill%2Ccs_srgb%2Cfl_progressive%2Cg_faces:center%2Cq_auto:good%2Cw_768/MTU3OTIzNTc4ODAwMjUyNTYy/brandenburg-gate-a-brief-historys-featured-photo.jpg"
+              alt="Card image cap"
+            />
+            <CardBody>
+              <CardTitle>Beispiel</CardTitle>
+              <CardSubtitle>Choose Correct Answer</CardSubtitle>
+
+              <Button outline color="info">
+                der
+              </Button>
+              <Button outline color="info">
+                die
+              </Button>
+              <Button outline color="info">
+                das
+              </Button>
+            </CardBody>
+          </Card>
+
+          <div className="deckButtons">
+            <Button outline color="info" className="previous">
+              previous
+            </Button>{" "}
+            <Button outline color="info" className="next">
+              {" "}
+              next
+            </Button>{" "}
+          </div>
+        </div>
+      </main>
+      <br />
+      <br /> <br />
+      {props.data &&
+        props.data.map((noun, index) => (
+          <ButtonCard key={index} name={noun.name} gender={noun.gender} />
+        ))}
```

Comment:
Using `index` as a React iteration "key" is not a good practice. It can lead to situations where items in the array are swallowed and other errors. Each "key" on each item needs to be fully unique. 

Right now I use some random generation plus a string in a function like this:

```
/**
 *
 * Generates a unique string for use in React "key" when iterating/mapping
 * Trying to avoid adding the Node UUID package to the app solely for use as a unique key in React
 *
 * @param {string} prefix - prefix for the unique string
 * @returns {string} - unique string
 */
export const generateKey = (prefix: string): string => {
  return `${prefix}_${Math.random().toString(36).substr(2, 9)}`;
};

```

Usage example:

```
<div css={locationPickerSuggestionsContainer}>
  {loading && <div>Loading...</div>}
  {suggestions.map((suggestion, index) => {
    return (
      <div key={generateKey(index)}>
        <span>{suggestion.description}</span>
      </div>
    );
  })}
</div>
```